### ISSUE TITLE
API response of the embed field could have description property as og:description?

### ISSUE THREAD

**Issue by tell-y** [prismicio/prismic-client#216]
Hi, I'm a newbie to the prismic development.
I don't know where should I report about it so I'm posting here.

I can not find about the embed filed in https://prismic.io/docs/technical-reference/prismicio-client
But there is in https://prismic.io/docs/technologies/react-template#embed
```
example_embed: {
  version: '1.0',
  url: 'https://prismic.io',
  type: 'link',
  title: 'Make your website editable for the whole team - Prismic',
  provider_name: null,
  thumbnail_url: 'https://images.prismic.io/prismic-website/6e49007fec52f99861047cdd55b3ab190ea04295_dev-landing-page-image.png?auto=compress,format',
  html: '<div data-type="unknown"><a href="https://prismic.io"><h1>Make your website editable for the whole team - Prismic</h1></div>',
  embed_url: 'https://prismic.io/',
}
```

I could expect the `title property` to be `og:title` so it should be able to have a `description property` as `og:description`.
I outputted a log of queried a result that has an embed link. That html had a description text inside.
I also checked the type but is not described about the description field.
```
// in @prismicio/types/dist/index.d.ts
declare type CommonEmbedData = {
    url?: string;
    version?: string;
    title?: string | null;
    html?: string | null;
    width?: number | null;
    height?: number | null;
    author_name?: string | null;
    author_url?: string | null;
    provider_name?: string;
    cache_age?: number | null;
    thumbnail_url?: string | null;
    thumbnail_width?: number | null;
    thumbnail_height?: number | null;
};
```

Thanks.

**Comment by angeloashmore** [comment]
Hi @tell-y, reporting this here is correct. 🙂 You can also report general TypeScript issues on the [`@prismicio/types`](https://github.com/prismicio/prismic-types) repository.

We can add a `description` field to `CommonEmbedData`. That makes sense to include to me. 👍

If you find there are other, more specific fields that you need and are not included in `CommonEmbedData`, you can override the types like this:

```ts
import * as prismicT from '@prismicio/types'

type CustomEmbedField = prismicT.CommonEmbedData & {
  description: string | null
}
```

Then you can use the type with type assertion:

```ts
const embedField = document.data.example_embed as CustomEmbedField
```

Or you can build your whole document type and use that when querying for your document:

```ts
import * as prismicT from '@prismicio/types'

type CustomEmbedField = prismicT.CommonEmbedData & {
  description: string | null
}

type PageDocument = prismicT.PrismicDocument<
  {
    example_embed: CustomEmbedData;
    another_example_field: prismicT.RichText
    // The rest of your fields…
  },
  'page'
>

const document = client.getByUID<PageDocument>('page', 'example')
```

I hope that helps! I will post here again once a `description` field is added to `CommonEmbedData`.

**Comment by angeloashmore** [comment]
@tell-y The `description` field is now added to `EmbedField`'s default set of fields. It is published as part of `@prismicio/types` `v0.1.23`.

You can update your project with:

```sh
npm update
```

Or if you just want to update your types, you can update/install `@prismicio/types` directly:

```
npm install @prismicio/types
# or
npm update @prismicio/types
```
